CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
 
+## Version 2.2.0
+- Includes all changes since 2.1.0 stable release.
```

Comment:
Changelog.md does not seem to have a 2.1.0 stable release as a reference point for 2.2.0 release.